README.md

```diff
@@ -1,108 +1,112 @@
+<br />
+<p align="center" style="background-color:white; border-radius:4px;">
+<img src="https://uploads-ssl.webflow.com/5f6353590bb01cacbcecfbac/62f21c38bc47b2d313fbf76d_Marchitecture%20-%20readme.svg" width="90%">
+</p>
+<br />
 
 <a name="quick"></a>
-### Installer - On-premise
-
-Follow the [installation instructions](https://hopsworks.readthedocs.io/en/stable/getting_started/installation_guide/platforms/hopsworks-installer.html).
-
-
-The installation takes roughly 1-2 hrs to complete, depending on your bandwidth. 
-
-
-### Installer - Azure CLI or GCP CLI
-
-Follow the [installation instructions](https://hopsworks.readthedocs.io/en/stable/getting_started/installation_guide/platforms/hopsworks-cloud-installer.html).
-
-If you have the Azure or GCP CLI utilities installed (on a Linux machine), then the Hopsworks-cloud-installer.sh will both install Hopsworks and provision the VMs in one command. 
-
-### Hopsworks IDE Plugin
-
-To work with the Hopsworks IDE plugin for IntelliJ/PyCharm, you can install it directly from the plugins menu of the IDE or [clone it](https://github.com/logicalclocks/hopsworks-ide-plugins) and follow the README.
-
-### Build with Maven
-```sh
-mvn install
-```
-Maven uses yeoman-maven-plugin to build both the front-end and the backend.
-Maven first executes the Gruntfile in the yo directory, then builds the back-end in Java.
-The yeoman-maven-plugin copies the dist folder produced by grunt from the yo directory to the target folder of the backend.
-
-You can also build Hopsworks without the frontend (for Java EE development and testing):
-```sh
-mvn install -P-web
-```
-
-<a name="mlops"></a>
-## Development and Operational ML on Hopsworks
-You can develop and run Python, Spark, and Flink applications on Hopsworks - in Jupyter notebooks, as jobs, or even notebooks as jobs. You can build production pipelines with the bundled Airflow, and even run ML training pipelines with GPUs in notebooks on Airflow. You can train models on as many GPUs as are installed in a Hopsworks cluster and easily share them among users.
+# 🚀 Quickstart
+
+## **APP - Serverless (beta)**
+### → **Go to [app.hopsworks.ai](https://app.hopsworks.ai)**
+Hopsworks is available as a serverless app, simply head to [app.hopsworks.ai](https://app.hopsworks.ai) and register with your **Gmail** or **Github** accounts. You will then be able to run a tutorial or access Hopsworks directly and try yourself. This is the prefered way to first experience the platform before diving into more advanced uses and installation requirements. 
+
+## **Azure, AWS & GCP**
+A [Managed Hopsworks](https://managed.hopsworks.ai) account is required as it is our platform for running Hopsworks and the Feature Store in the cloud. It integrates seamlessly with third party platforms such as Databricks, SageMaker and KubeFlow.
+
```

Comment:
I think this sentence can be improved. I would state immediately that managed.hopsworks.ai integrates with the customer AWS/Azure/GCP environment.